From e6049a74455d14e08628c40dfa76be99ad0e11c1 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Tue, 30 Jul 2019 15:39:18 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399: Change minimum voltage to 825mV

It is better to make the voltage greater than 810mV and it will be more
stable.

Change-Id: I4f9294a0629a8c3b83e930f0dec20d939130070d
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-opp.dtsi | 72 ++++++++++----------
 1 file changed, 36 insertions(+), 36 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-opp.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-opp.dtsi
index cd72acd304ec..a17ec0b1f375 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-opp.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-opp.dtsi
@@ -42,20 +42,20 @@
 
 		opp-408000000 {
 			opp-hz = /bits/ 64 <408000000>;
-			opp-microvolt = <800000 800000 1250000>;
-			opp-microvolt-L0 = <800000 800000 1250000>;
-			opp-microvolt-L1 = <800000 800000 1250000>;
-			opp-microvolt-L2 = <800000 800000 1250000>;
-			opp-microvolt-L3 = <800000 800000 1250000>;
+			opp-microvolt = <825000 825000 1250000>;
+			opp-microvolt-L0 = <825000 825000 1250000>;
+			opp-microvolt-L1 = <825000 825000 1250000>;
+			opp-microvolt-L2 = <825000 825000 1250000>;
+			opp-microvolt-L3 = <825000 825000 1250000>;
 			clock-latency-ns = <40000>;
 		};
 		opp-600000000 {
 			opp-hz = /bits/ 64 <600000000>;
-			opp-microvolt = <800000 800000 1250000>;
-			opp-microvolt-L0 = <800000 800000 1250000>;
-			opp-microvolt-L1 = <800000 800000 1250000>;
-			opp-microvolt-L2 = <800000 800000 1250000>;
-			opp-microvolt-L3 = <800000 800000 1250000>;
+			opp-microvolt = <825000 825000 1250000>;
+			opp-microvolt-L0 = <825000 825000 1250000>;
+			opp-microvolt-L1 = <825000 825000 1250000>;
+			opp-microvolt-L2 = <825000 825000 1250000>;
+			opp-microvolt-L3 = <825000 825000 1250000>;
 			clock-latency-ns = <40000>;
 		};
 		opp-816000000 {
@@ -63,8 +63,8 @@
 			opp-microvolt = <850000 850000 1250000>;
 			opp-microvolt-L0 = <850000 850000 1250000>;
 			opp-microvolt-L1 = <825000 825000 1250000>;
-			opp-microvolt-L2 = <800000 800000 1250000>;
-			opp-microvolt-L3 = <800000 800000 1250000>;
+			opp-microvolt-L2 = <825000 825000 1250000>;
+			opp-microvolt-L3 = <825000 825000 1250000>;
 			clock-latency-ns = <40000>;
 			opp-suspend;
 		};
@@ -133,20 +133,20 @@
 
 		opp-408000000 {
 			opp-hz = /bits/ 64 <408000000>;
-			opp-microvolt = <800000 800000 1250000>;
-			opp-microvolt-L0 = <800000 800000 1250000>;
-			opp-microvolt-L1 = <800000 800000 1250000>;
-			opp-microvolt-L2 = <800000 800000 1250000>;
-			opp-microvolt-L3 = <800000 800000 1250000>;
+			opp-microvolt = <825000 825000 1250000>;
+			opp-microvolt-L0 = <825000 825000 1250000>;
+			opp-microvolt-L1 = <825000 825000 1250000>;
+			opp-microvolt-L2 = <825000 825000 1250000>;
+			opp-microvolt-L3 = <825000 825000 1250000>;
 			clock-latency-ns = <40000>;
 		};
 		opp-600000000 {
 			opp-hz = /bits/ 64 <600000000>;
-			opp-microvolt = <800000 800000 1250000>;
-			opp-microvolt-L0 = <800000 800000 1250000>;
-			opp-microvolt-L1 = <800000 800000 1250000>;
-			opp-microvolt-L2 = <800000 800000 1250000>;
-			opp-microvolt-L3 = <800000 800000 1250000>;
+			opp-microvolt = <825000 825000 1250000>;
+			opp-microvolt-L0 = <825000 825000 1250000>;
+			opp-microvolt-L1 = <825000 825000 1250000>;
+			opp-microvolt-L2 = <825000 825000 1250000>;
+			opp-microvolt-L3 = <825000 825000 1250000>;
 			clock-latency-ns = <40000>;
 		};
 		opp-816000000 {
@@ -154,8 +154,8 @@
 			opp-microvolt = <825000 825000 1250000>;
 			opp-microvolt-L0 = <825000 825000 1250000>;
 			opp-microvolt-L1 = <825000 825000 1250000>;
-			opp-microvolt-L2 = <800000 800000 1250000>;
-			opp-microvolt-L3 = <800000 800000 1250000>;
+			opp-microvolt-L2 = <825000 825000 1250000>;
+			opp-microvolt-L3 = <825000 825000 1250000>;
 			clock-latency-ns = <40000>;
 			opp-suspend;
 		};
@@ -235,27 +235,27 @@
 
 		opp-200000000 {
 			opp-hz = /bits/ 64 <200000000>;
-			opp-microvolt = <800000>;
-			opp-microvolt-L0 = <800000>;
-			opp-microvolt-L1 = <800000>;
-			opp-microvolt-L2 = <800000>;
-			opp-microvolt-L3 = <800000>;
+			opp-microvolt = <825000>;
+			opp-microvolt-L0 = <825000>;
+			opp-microvolt-L1 = <825000>;
+			opp-microvolt-L2 = <825000>;
+			opp-microvolt-L3 = <825000>;
 		};
 		opp-300000000 {
 			opp-hz = /bits/ 64 <300000000>;
-			opp-microvolt = <800000>;
-			opp-microvolt-L0 = <800000>;
-			opp-microvolt-L1 = <800000>;
-			opp-microvolt-L2 = <800000>;
-			opp-microvolt-L3 = <800000>;
+			opp-microvolt = <825000>;
+			opp-microvolt-L0 = <825000>;
+			opp-microvolt-L1 = <825000>;
+			opp-microvolt-L2 = <825000>;
+			opp-microvolt-L3 = <825000>;
 		};
 		opp-400000000 {
 			opp-hz = /bits/ 64 <400000000>;
 			opp-microvolt = <825000>;
 			opp-microvolt-L0 = <825000>;
 			opp-microvolt-L1 = <825000>;
-			opp-microvolt-L2 = <800000>;
-			opp-microvolt-L3 = <800000>;
+			opp-microvolt-L2 = <825000>;
+			opp-microvolt-L3 = <825000>;
 		};
 		opp-600000000 {
 			opp-hz = /bits/ 64 <600000000>;
-- 
2.35.3

